commodity_trading/demo/purchase_order_line.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="purchase_order_line_14" model="purchase.order.line">
+        <field name="product_id" ref="product_product_1"/>
+        <field name="order_id" ref="purchase_order_1"/>
+        <field name="product_qty">125.0</field>
+        <field name="price_unit">123.0</field>
```

Comment:
price_unit comes should be set in the product. If already set, we need to remove it